docs/units/equivalencies.rst

```diff
@@ -463,15 +463,20 @@ point of 3631.1 Jy::
 Temperature Equivalency
 -----------------------
 
-The :func:`~astropy.units.temperature` equivalency allows conversion
-between the Celsius, Fahrenheit, Rankine and Kelvin.
+Conversions between kelvins and degrees Rankine do not require using any
+equivalencies because the Kelvin scale and the Rankine scale have the same zero
+point::
 
-Example
-^^^^^^^
-
-.. EXAMPLE START: Using the Temperature Equivalency
```

Comment:
These examples were put in because it helps indexing them for `learn.astropy`. To be honest, I'm not sure that is still being done, but removing those comments would seem out of scope here.

Reply:
Removing that did not cause any errors or warnings when building the documentation neither locally nor in the CI job and the rendered documentation looks like intended, so it does not fulfill any obvious purpose. If there were some non-obvious purpose for keeping it then it would surely be documented at https://github.com/astropy/astropy/blob/d816139637181c4f3f1997bc46c775e7f2ea4d02/docs/development/docguide.rst, but it's not even mentioned there at all. Apparently this is some obsolete leftover of some previous tool that isn't important anymore.

Reply:
As I noted, it was done for `learn.astropy`, and someone was paid to put all those examples in. Simplest by far would be to just leave it as is, but let me ping @kelle to see whether she knows whether these `.. EXAMPLE START` are still being used.

Generally, though, I think one should not make changes that are unrelated to the PR.

Reply:
Until we hear from @astropy/learn-astropy-team, I agree, let's leave the markers in the docs.

Reply:
I am editing the lines immediately surrounding this, so just leaving it in place is not safe either.

Reply:
I'll rephrase what I said before. You are telling me to leave it in place, but the place to leave it in no longer exists.